src/properties/mod.rs

```diff
@@ -67,51 +67,73 @@ macro_rules! impl_read {
     () => {
         #[inline]
         pub(crate) fn read<R: Read + Seek>(
-            cursor: &mut R,
+            reader: &mut R,
             include_header: bool,
         ) -> Result<Self, Error> {
             if include_header {
-                Self::read_header(cursor)
+                Self::read_header(reader)
             } else {
-                Self::read_body(cursor)
+                Self::read_body(reader)
             }
         }
     };
 
     (options) => {
         #[inline]
         pub(crate) fn read<R: Read + Seek>(
-            cursor: &mut R,
-            options: &mut PropertyOptions,
+            reader: &mut R,
             include_header: bool,
+            options: &mut PropertyOptions,
         ) -> Result<Self, Error> {
             if include_header {
-                Self::read_header(cursor, options)
+                Self::read_header(reader, options)
             } else {
-                Self::read_body(cursor, options)
+                Self::read_body(reader, options)
             }
         }
     };
 }
 
 /// The default read_header() implementation
 macro_rules! impl_read_header {
```

Comment:
Please add more docs to this and the write macros so that new contributors can get up to speed with it quicker.